scripts/ansible/roles/golang_base/tasks/ubuntu-20.04.yml

```diff
@@ -0,0 +1,7 @@
+- name: Install Go
+  become: true
+  apt:
+    name:
+      - golang-go
```

Comment:
Does this clearly identify the version?

Reply:
On Ubuntu 20.04 the role uses a core package, so it wont change from 1.13.x.

On Ubuntu 18.04 I went with [ppa:longsleep/golang-backports](https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports), so today it will install golang 1.15